datasetinsights/datasets/__init__.py

```diff
@@ -12,6 +11,4 @@
     "GroceriesReal",
     "GoogleGroceriesReal",
     "NyuDepth",
-    "SynDetection2D",
```

Comment:
[mustfix] If we remove `SynDetection2D`, it cannot find `SynDetection2D` class in `Dataset.find()`.

Reply:
To keep SynDetection2D I need to import synthetic, which is causing cycle.